src/Appwrite/Platform/Modules/DevKeys/Http/DevKeys/DeleteKey.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Appwrite\Platform\Modules\DevKeys\Http\DevKeys;
```

Comment:
We don't need to have the same namespace `DevKeys` twice. Also, arguably the module name here should be projects and not devkeys. When we have nesting we don't need a folder for it. We can try and replicated the CRUD in this PR: https://github.com/appwrite-labs/cloud/pull/1394